lgsm/functions/update_vintagestory.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Update module looks good (and works well), however, does not follow the same convention as other update modules. For consistency (to make it easier to do updates to all update modules in future) please can you re-arrange and update function, variable names and general convention (as much as possible) to be similar to others like update_minecraft.sh.

Also the output looks different from the standard

For example
```
[  OK  ] Updating cssserver: Checking for update: SteamCMD

No update available
* Local build: 4259010
* Remote build: 4259010
* Branch: public
https://steamdb.info/app/232330/
```

Compared to 
```
[  OK  ] Updating vintsserver: Checking for update: vintagestory.at

No update available
* Local build: 1.14.2
```

Thank you 👍

Reply:
Renamed variables to the convention.
Would you like a full restructure for the module here ?